### DISCUSSION TITLE
CRNN Model Onnx conversion

### DISCUSSION THREAD

**Discussion by Shiva-Ai4m** [mindee/doctr#1543] [General]
Hi @felixT2K @felixdittrich92  , i am unable to convert crnn recognition  model  to onnx model. Can you please help me in that or send a the file ?

**Comment by felixdittrich92** [comment]
Hi @Shivashankarar :wave:,

What's the error / issue ? Have you followed https://mindee.github.io/doctr/using_doctr/using_model_export.html#export-to-onnx ? :)
